python/code_to_gast/py_helpers.py

```diff
@@ -86,8 +86,9 @@ def pyop_to_str(op):
         ast.GtE: ">=",
         ast.Lt: "<",
         ast.LtE: "<=",
-        ast.Eq: "=="
-
+        ast.Eq: "==",
+        ast.UAdd: "++",
```

Comment:
I thought `++` and `--` aren't supported in python?

From my brief and very likely mistaken googling it looks like ast.UAdd might correspond with `+`. :)
https://www.programcreek.com/python/example/61006/ast.UAdd

Reply:
After doing more research I was wrong that `++a` works in Python. It compiles on the Python ast parser but does not seem to be actually allowed in Python